### ISSUE TITLE
[Dell]: [Vostro 5515] [Unlocks but unable to remove the admin password that is set]

### ISSUE THREAD

**Issue by rashadmughal** [bacher09/pwgen-for-bios#151]
The code on https://bios-pw.org/ works fine for 'Unlock Setup', I am then able to change the any bios settings.

The code is generated using the hash suffixed with the service tag e.g.: a2852d96ede42bc0-7KYQQ93 

However, I am then UNABLE to REMOVE the admin password, when I attempt to remove it and put in the master password as the old password, it simply states incorrect password. I have tried CTRL + ENTER + ENTER and still doesn't unset the password.

TL;DR
Master bios password unlocks bios - but doesn't remove/unset password.

**Comment by K3YL0** [comment]
I have some software for this, I'll have to dig up!

**Comment by K3YL0** [comment]
Also, Have you tried to boot into the BIOS Menu after entering the password?
